### PR TITLE
Update tonic

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
Generally that is the duplicate of this PR https://github.com/input-output-hk/jormungandr/pull/3676, but without the counters update.

### CODE REVIEW

File: jormungandr-lib/Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ typed-bytes = { git = "https://github.com/input-output-hk/chain-libs.git", branc
 rand = "0.8"
 rand_core = "0.6"
 rand_chacha = "0.3"
-chrono = { version = "0.4", features = ["serde"] }
+chrono = { version = "0.4.19", features = ["serde"] }
```
Comment (enzoc4) [inline jormungandr-lib/Cargo.toml:24]: this wasn't part of the other PR, right?

**Review by enzoc4 — APPROVED** [review]
LGTM, and the other PR was already approved. I guess it will have to be rebased, or we just merge this one into the fund7 branch instead of master. But probably rebasing it is better.

File: jormungandr-lib/Cargo.toml (hunk 1 above)
Comment (Mr-Leshiy) [reply jormungandr-lib/Cargo.toml:24]: no, it was not

**Comment by Mr-Leshiy** [comment]
@enzoc4 I am considering to merge it into the master and also merge into the master this PR https://github.com/input-output-hk/jormungandr/pull/3676, and after that create a new branch fund7 and revert update related to the counters. So the main point is to keep the current master is up to date with all features that we have there and updates.

**Pull request merged**